ntbs-service/appsettings.Development.json

```diff
@@ -9,6 +9,9 @@
     "UseDummyAuth": true
   },
   "ExternalLinks": {
-      "ReportingUri": "" 
-  } 
+    "ReportingUri": ""
+  },
+  "Environment": {
+    "Name": "Dev"
+  }
```

Comment:
I think I would be tempted to use a more explicit name here, `MigrationTablePrefix` for example - the current setup makes it a little confusing, since `Environment` is already an established env value!